aggregator/src/aggregator/accumulator.rs

```diff
@@ -31,8 +31,10 @@ use std::{
 pub struct Accumulator<
     const SEED_SIZE: usize,
     Q: AccumulableQueryType,
-    A: vdaf::Aggregator<SEED_SIZE, 16>,
-> {
+    A: vdaf::Aggregator<SEED_SIZE>,
+> where
+    for<'a> &'a A::AggregateShare: Into<Vec<u8>>,
```

Comment:
Done, but tbh, I disagree somewhat: I think it's OK/reads better to have the trait bound that defines what the type "is" (`vdaf::Aggregator` in this case) in the parameter list, with additional "properties" trait bounds (such as `Into<Vec<u8>>`, in this case) in the `where` clause.

This makes the generic parameters more easily readable/a summary of what the types are; if we stick them in the `where` clause this can be harder to parse out if there are quite a few `where` clauses.

(this is an entirely stylistic argument, of course, so I'm happy going with whatever folks are more comfortable with.)